src/lib/forward.ts

```diff
@@ -74,20 +77,32 @@ export class EventForwarder {
         })
       )
     );
+
     this.clients.driver.controller.on("heal network progress", (progress) =>
       this.forwardEvent({
         source: "controller",
         event: "heal network progress",
         progress: Object.fromEntries(progress),
       })
     );
+
     this.clients.driver.controller.on("heal network done", (result) =>
       this.forwardEvent({
         source: "controller",
         event: "heal network done",
         result: Object.fromEntries(result),
       })
     );
+
+    this.clients.driver.controller.on(
+      "statistics updated",
+      (statistics: ControllerStatistics) =>
+        this.forwardEvent({
+          source: "controller",
+          event: "statistics updated",
+          statistics: Object.freeze(statistics),
```

Comment:
Why do you need this to keep TypeScript happy, what was the error ?

Reply:
And if you don't do anything, just do `statistics as TypeItShouldBe`. Although you disable typechecking then.

Reply:
I get the following error:
```
Type 'ControllerStatistics' is not assignable to type 'JSONValue'.
  Type 'ControllerStatistics' is not assignable to type '{ [key: string]: JSONValue; }'.
    Index signature is missing in type 'ControllerStatistics'.ts(2322)
outgoing_message.ts(23, 3): The expected type comes from this index signature.
```

What type should I use? I tried JSONValue but I get the following:
```
Conversion of type 'ControllerStatistics' to type 'JSONValue' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Type 'ControllerStatistics' is not comparable to type '{ [key: string]: JSONValue; }'.
    Index signature is missing in type 'ControllerStatistics'.ts(2352)
```

Reply:
You can do `as unknown` or `as any`.

Reply:
didn't like `as unknown` without recasting it to `JSONValue` but `as any` worked 👍🏾